decidim-core/config/locales/en.yml

```
@@ -830,6 +833,25 @@ en:
         note: You received this email because you're subscribed to newsletters on %{organization_name}. You can change your settings on your <a href="%{link}">notifications page</a>.
         see_on_website: Can’t see this email correctly? View it on the <a href="%{link}" target="_blank">website</a>.
         unsubscribe: To opt out of receiving this type of email, <a href="%{link}" target="_blank" class="unsubscribe">Unsubscribe</a>.
+    newsletter_templates:
+      basic_only_text:
+        body_preview: body
+        name: Basic (only text)
+      basic_only_text_settings_form:
+        body: Body
+        interpolations_hint: 'Hint: You can use "%{name}" anywhere in the body or subject and it will be replaced by the recipient''s name.'
+      image_text_cta:
+        body_preview: body
+        cta_text_preview: Call To Action text
+        introduction_preview: introduction
```

The newsletter template gallery feature is great 👏, a more realistic example text would make it stand out even more. A _Lorem Ipsum_ dummy text or some real text:

```suggestion
        introduction_preview: "Decidim is a Free Open-Source participatory democracy platform for cities and organizations. But Decidim is more than a digital platform: it’s a common's free and open project and infrastructure involving code, documentation, design, training courses, a legal framework, collaborative interfaces, user and facilitation communities, and a common vision."  
```